pstoedit: add page (#10939)
tldr-pages/tldr: pages/linux/pstoedit.md

# pstoedit

> Convert PDF files into various image formats.

> > More information: <http://www.pstoedit.net/>.

Reviewer: ```suggestion
> More information: <http://www.pstoedit.net>.
```

> - Convert a PDF page to PNG or JPEG [f]ormat:

Reviewer 2: ```suggestion
- Convert a PDF page to PNG or JPEG format:
```
The long option doesn't seem to be present in the [manual](http://www.calvina.de/pstoedit/pstoedit.htm)

> `pstoedit -page {{page_number}}  -f magick {{filename}}  page.png|page.jpg`

Reviewer 2: ```suggestion
`pstoedit -page {{page_number}}  -f magick {{path/to/file.pdf}}  {{page.png|page.jpg]}}`
```

> `pstoedit -page {{page_number}}  -f magick {{path/to/file.pdf}}  {{page.png|page.jpg]}}`

Reviewer: ```suggestion
`pstoedit -page {{page_number}} -f magick {{path/to/file.pdf}} {{page.png|page.jpg]}}`
```

- Convert multiple PDF pages to numbered images:

> `pstoedit -f magick {{filename}} page%d.png|page%d.jpg`

Reviewer 2: ```suggestion
`pstoedit -f magick {{path/to/file}} {{page%d.png|page%d.jpg}}`
```
